compute/instances/reference-content/preventing-outgoing-ddos.mdx

```diff
@@ -10,29 +10,28 @@ categories:
   - compute
 ---
 
-
 <Message type="important">
-  The explanations given below are known best-practices. They do not guarantee that your resources will not be locked if we detect that they are part of a DDoS attack.
+  The explanations given below are known best practices. They do not guarantee that your resources will not be locked if we detect that they are part of a DDoS attack.
 </Message>
 
 ## DoS Overview
 
-Denial of Service (DoS) attack is an attack through which a person can render a system unusable, or significantly slow it down for legitimate users, by overloading its resources.
+A Denial of Service (DoS) attack is an attack through which a person can render a system unusable, or significantly slow it down for legitimate users, by overloading its resources.
 
 [A Distributed Denial of Service (DDoS) attack](DDoS_Quick_Guide.pdf) is a DoS attack that is performed at the same time by a multitude of compromised systems.
 
-The goal of a DoS is not to gain unauthorized access to machines or data but to prevent legitimate users of a service from using it.
+The goal of a DoS is not to gain unauthorized access to machines or data, but to prevent legitimate users of a service from using it.
 
 You are responsible for your resources.
-If a resource you control takes part in a DDoS you will be considered responsible for attacking the target of this DDoS.
+If a resource you control takes part in a DDoS, you will be considered responsible for attacking the target of this DDoS.
 
 Scaleway will lock any resources (e.g. Instances, Kubernetes clusters, Elastic Metal servers) that are identified as a contributor to a DDoS.
 This lock can be done without prior notice to protect our network and the target network; this is written in our Terms and Conditions ([Scaleway](https://www.scaleway.com/en/terms/), [Online](https://www-uploads.scaleway.com/cgv_en_36883f3024.pdf)).
 
 ## Preventing Memcache from being used in a DDoS attack
 
 [Memcached](https://memcached.org/) is a free & open-source, high-performance, distributed memory object caching system.
-It is used as a key-value store in memory.
+It is used as a key-value stored in memory.
```

Comment:
```suggestion
It is used as a key-value store in memory.
```
[key-value store](https://www.scylladb.com/glossary/key-value-database/) is used a noun here ;)

Reply:
It's ok. I was unsure about this one. Thanks.